src/privateStore.js

```diff
@@ -26,6 +26,13 @@ class PrivateStore extends KeyValueStore {
     return super.set(key, value)
   }
 
+  async setMultiple (keys, values) {
+    utils.throwIfNotEqualLenArrays(keys, values)
+    const dbKeys = keys.map(this._genDbKey, this)
+    const encryptedValues = values.map(this._encryptEntry, this)
```

Comment:
Cool, didn't know you could pass `this` like that. I've always just used `.bind`